golang/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM golang:1.11
+FROM golang:1.12
```

Comment:
This should at least be `1.12.8` because of the security bugs fixed in that version https://github.com/golang/go/issues?q=milestone%3AGo1.12.8

BUT why not just use `latest`? currently it is `1.12.8`

Reply:
latest could be scary... We really only need `1.12.8`